server/events/index.js

```diff
@@ -74,8 +75,21 @@ class RoomEventBridge extends EventEmitter {
     const clientSocket = this._getClientSocketInRoom(
       this._userSocketIdMap[userId]
     );
+    let otherPlayers = [];
```

Comment:
@existentialcoder , @mahadevans87 : Instead of sending the hint to everyone except the current drawing user, can we just broadcast it to everyone and handle if we need to show hint or not at the client side ? What do you guys think ?

Reply:
My only issue with that is that then someone can manipulate the logic. The whole point is that we shouldn't reveal the word in the client side right?

Reply:
@mahadevans87 : I agree. But we are not sending the entire word to clients. We are just sending the hint right? Something like ` A _ P_E`

Reply:
Also, @existentialcoder : when we use `io.sockets`, does it return all sockets irrespective of the room in which the user joined ? I understand currently there is only one room. But when two rooms are there, we should not send a hint to all sockets ( players from both rooms).